Main_Control_Node/src/LCD/LCD.c

```diff
@@ -240,6 +240,19 @@ void OutputPaddedInteger(int32_t num, uint8_t dec, uint8_t x, uint8_t y)
         str[0] = '-';
         num = -1 * num;
     }
+	else if (num == 0)
+	{
+		str[3] = '0';
+	}
+	
+	if (num > 999)
+	{
+		num = 999;
+	}
+	else if (num < -999)

```

Comment:
Is this necessary after what happens in line 238?

Reply:
oof yeah i'm dumb